go-packages/meep-gis-asset-mgr/asset-mgr.go

```diff
@@ -42,8 +42,7 @@ const (
 // Enable profiling
```

Comment:
General comment on asset-mgr.go
- Profiling code is enabled & will impact execution on the production setups
- I recommend disabling this unless we are chasing an issue in production